snapshot/generate_test.go

```diff
@@ -118,6 +149,10 @@ func hashData(input []byte) common.Hash {
 
 // Tests that snapshot generation with existent flat state.
 func TestGenerateExistentState(t *testing.T) {
+	if true {
+		enableLog()
+	}
```

Comment:
why do we need this condition?

Reply:
This is not necessary, I will remove this.